lib/shared/bucketing-assembly-script/assembly/types/eventQueueOptions.ts

```diff
@@ -1,13 +1,7 @@
 import { JSON } from 'assemblyscript-json/assembly'
-import {
-    getF64FromJSONValue,
-    getI32FromJSONValue
-} from '../helpers/jsonHelpers'
+import { getI32FromJSONValue } from '../helpers/jsonHelpers'
 
 export class EventQueueOptions extends JSON.Obj {
-    flushEventsMS: f64 = 10 * 1000
```

Comment:
none of these options were used in AS.

Reply:
shouldn't we be handling the disabling of event logging in AS?

Reply:
it can be, currently its handled on the Native side.

Reply:
https://taplytics.atlassian.net/browse/DVC-4408